src/start_server.ts

```diff
@@ -456,6 +468,23 @@ async function startWithFirstAvailablePort(
 async function tryStartWithPort(
     options: ServerOptions, app: express.Application, port: number) {
   const server = await createServer(app, options);
+
+  if (options.liveReloadPath) {
+    const wss = new WebSocketServer({server: server});
+    wss.on('connection', (ws: WebSocket) => {
+      const watcher = watchFile(options.liveReloadPath);
```

Comment:
As written I think this creates another file watcher every time the browser reloads. Depending on how watchFile is implemented it may be much more efficient to create the watcher once and just add and remove event listeners as sockets open and close.